srcpkgs/phpMyAdmin/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'phpMyAdmin'
 pkgname=phpMyAdmin
-version=5.0.4
+version=5.2.0
 revision=1
 wrksrc="phpMyAdmin-${version}-all-languages"
 conf_files="/etc/webapps/phpMyAdmin/config.inc.php"
-depends="php mysql"
+depends="php mariadb"
 short_desc="Web interface for MySQL and MariaDB"
-maintainer="Franc[e]sco <lolisamurai@tfwno.gf>"
+maintainer="William Desportes <williamdes@wdes.fr>"
```

Comment:
@Francesco149 can they adopt this from you?

Reply:
ok to adopt, I am not maintaining my void packages these days